### ISSUE TITLE
Lightmasts (tower:type lighting) are rendered as communication mast 

### ISSUE THREAD

**Issue by unixhex** [der-stefan/OpenTopoMap#311]
Lightmasts  (man_made mast, tower:type lighting) are rendered with the symbol for communication mast (man_made mast, tower:type communication)

Examples: 
https://opentopomap.org/#marker=17/48.1121798/11.6355840
in OpenStreetmap is this node:
https://www.openstreetmap.org/node/4854047872

or
https://opentopomap.org/#marker=17/48.1127755/11.6384082
in Openstreetmap:
https://www.openstreetmap.org/node/4854047875

**Comment by der-stefan** [comment]
Good catch! There will be no update the the outdated raster tiles, but I'll consider this for the upcoming vector tiles.
